### PR TITLE
Adds new delete row modal

### PR DESCRIPTION
**Pull request by javierarce**
Similar to the delete column modal, this PR introduces the brand new delete row modal: 

screen shot 2015-06-17 at 18 44 30

(as a bonus, this PR also fixes the filename of the delete_column_view.spec.js)

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4409/))

**Comment by javierarce** [comment]
@viddo: can you apply your x-ray vision to this PR, please?

File: lib/assets/javascripts/cartodb/table/row_header_dropdown.js [hunk 1]
```diff
@@ -85,8 +85,20 @@ cdb.admin.RowHeaderDropdown = cdb.admin.DropdownMenu.extend({
...
     var self = this;
+
     this.killEvent(e);
 
+    if (this.options.user.featureEnabled('new_modals')) {
+
+      var view = new cdb.editor.DeleteRowView({
+        table: this.table,
+        row: this.row
+      });
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/table/row_header_dropdown.js]: I think these attrs should be provided too: ```
clean_on_hide: true,
enter_to_confirm: true
``` 
that way the modal is removed & cleaned if cancelled

File: lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  ok: function() {
+    var self = this
+    this._panes.active('loading');
+    this.table.trigger('removing:row');
+    this.row.destroy({
+      success: function() {
+        self.table.trigger('remove:row', self.row);
+      }
+    });
+    this.clean();
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js]: I think w/ the fix of [my prev comment](https://github.com/CartoDB/cartodb/pull/4103/files#r32649781) this clean call should not be necessary.

File: lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  },
+
+  ok: function() {
+    var self = this
+    this._panes.active('loading');
+    this.table.trigger('removing:row');
+    this.row.destroy({
+      success: function() {
+        self.table.trigger('remove:row', self.row);
+      }
```
Comment (viddo) [inline lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js]: What happens when the destroy fails?

File: lib/assets/javascripts/cartodb/table/row_header_dropdown.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/table/row_header_dropdown.js]: :ok_hand:

File: lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/common/dialogs/delete_row/delete_row_view.js]: :ok_hand:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4415/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4416/))

**Comment by javierarce** [comment]
retest this please

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4461/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4462/))

**Pull request merged**